addons/stock_account/static/src/stock_valuation/controller.js

```diff
@@ -22,7 +22,7 @@ export class StockValuationReportController {
 
     async loadReportData() {
         const kwargs = {
-            date: this.state.date.toFormat("yyyy-MM-dd"),
+            date: this.state.date.toFormat("yyyy-MM-dd", { numberingSystem: "latn" }),
```

Comment:
```suggestion
            date: serializeDate(this.state.date),
```
You should use `serializeDate()` from `web/core/l10n/dates`. It handles the `numberingSystem` as well, and it's a more global fix of a recurring issue 🙂